src/deps/system_contracts.rs

```diff
@@ -103,6 +108,11 @@ pub static COMPILED_IN_SYSTEM_CONTRACTS: Lazy<Vec<DeployedContract>> = Lazy::new
             CREATE2_FACTORY_ADDRESS,
             include_bytes!("contracts/Create2Factory.json").to_vec(),
         ),
+        (
+            "TimestampAsserter",
+            TIMESTAMP_ASSERTER_ADDRESS,
```

Comment:
How does `era_test_node` know to use this newly deployed contract for timestamp assertions? Or is this address something a user needs to include in their requests?

Reply:
The latter - the user would use this address in their tests whenever they are calling timestamp asserter.

Reply:
Do we need to expose this address somehow so users can do that? Perhaps include it in the Readme or print it out to the console?

Reply:
We can defer for now as ideally we would want to expose this address in a manner that's consistent with how Era would do it. That address looks to be distributed by word of mouth at the moment